### PR TITLE
Add is_explicit flag to MusicRecording

### PR DESCRIPTION
**Pull request by gauss0818**

### CODE REVIEW

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java [hunk 1]
```diff
@@ -33,18 +33,23 @@ public class MusicRecording {
   private final MusicRelease musicRelease;
   private final List<MusicGroup> byArtists;
 
+  // Whether the Music entity contains objectionable language.
```
Comment (jzacsh) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java:36]:
```suggestion
  /* Whether the Music entity contains objectionable language, according to the providing adapter's service policies. */
```

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java [hunk 2]
```diff
@@ -67,6 +72,10 @@ public List<MusicGroup> getByArtists() {
     return byArtists;
   }
 
+  public boolean getIsExplicit() {
```
Comment (jzacsh) [inline portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java:75]: nit: wdyt of moving the javadoc here onto this public member?

**Review by jzacsh — CHANGES_REQUESTED** [review]

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java (hunk 2 above)
Comment (gauss0818) [reply portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java:74]: That makes more sense. Thank you for the advice!

File: portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java (hunk 1 above)
Comment (gauss0818) [reply portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicRecording.java]: Move the comment to public member.

File: extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java [hunk 3]
```diff
@@ -280,7 +280,8 @@ private MusicPlaylistItem convertPlaylistItem(
                 release.getIcpn(),
                 release.getTitle(),
                 createMusicGroups(release.getArtists())),
-            createMusicGroups(track.getArtists())),
+            createMusicGroups(track.getArtists()), false),
+        // TODO: Add explicit flag to GooglePlaylistItem
```
Comment (jzacsh) [inline extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java:284]: is this a nice-to-have at some point or pre-launch? (should we use the other TODO-format we were using?)

**Review by jzacsh — APPROVED** [review]

File: extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java (hunk 3 above)
Comment (gauss0818) [reply extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java]: It's pre-launch. Updated the TODO format.

**Review by jzacsh — APPROVED** [review]

**Pull request merged**